app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -480,6 +490,16 @@ export class ModuleSearch extends React.PureComponent<Props, State> {
                                         value={search}
                                         onChange={this.onChange}
                                     />
+                                    {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+                                    <span
```

Comment:
Maybe instead of `eslint-disable` using a `<button>` would be a better direction here. It does feel kinda buttony to me. ;)

Reply:
👍